packages/api/src/routes/medical/dtos/facilityDTO.ts

```diff
@@ -9,13 +10,26 @@ export type FacilityDTO = BaseDTO & {
   tin: string | undefined;
   active: boolean | undefined;
   address: AddressStrictDTO;
+  cqType: FacilityType;
+  cqActive: boolean | undefined;
+  cqOboOid: string | null;
+  cwType: FacilityType;
+  cwActive: boolean | undefined;
+  cwOboOid: string | null;
 };
 
 export function dtoFromModel(facility: Facility): FacilityDTO {
   const { name, npi, tin, active, address } = facility.data;
+  const { cqType, cqActive, cqOboOid, cwType, cwActive, cwOboOid } = facility;
   return {
     ...toBaseDTO(facility),
     id: facility.id,
+    cqType,
+    cqActive,
+    cqOboOid,
+    cwType,
+    cwActive,
+    cwOboOid,
```

Comment:
We want to start exposing these fields to our cxs? AFAIK we dont

Reply:
Moved to the `/cx-data` endpoint.